pyvecorg/build.py

```diff
@@ -40,14 +53,14 @@ def generate_yaml(data):
 
 if __name__ == '__main__':
     # Build data/members_list.yml and avatar images
-    response = requests.get(MEMBERS_CSV_URL)
-    response.raise_for_status()
-
-    members = [
-        member for member
-        in parse_members_csv(response.content.decode('utf-8'))
-        if member.get('role')  # currently we only display members with role
-    ]
+    gsa_path = PACKAGE_DIR / 'google_service_account.json'
+    gsa_json = os.getenv('GOOGLE_SERVICE_ACCOUNT') or gsa_path.read_text()
+    gsa = json.loads(gsa_json)
+
+    doc_key = '1n8hzBnwZ5ANkUCvwEy8rWsXlqeAAwu-5JBodT5OJx_I'
+    rows = read_spreadsheet(doc_key, 'list', gsa)
+    members = [member for member in parse_members(rows)
```

Comment:
can we get some fallback here in case I want to hack other part of the website and when I don't want to obtain the credentials?